frontend/src/v5/ui/components/shared/drawingRevisionDetails/drawingRevisionDetails.component.tsx

```diff
@@ -0,0 +1,144 @@
+		.sort((a, b) => b.timestamp.getTime() - a.timestamp.getTime());
+	const selected = revisions.findIndex((r) => !r.void);
+
+	const handleDownloadRevision = (revisionId) => {
+		window.location.href = getRevisionFileUrl(teamspace, project, drawingId, revisionId);
+	};
+
+	useEffect(() => {
+		if (!revisions.length) {
+			DrawingRevisionsActionsDispatchers.fetch(teamspace, project, drawingId);
+		}
+	}, []);
+
+	if (!revisionsCount && !isLoading && !revisions.length) {
+		return (
+			<RevisionsListEmptyWrapper>
+				<RevisionsListEmptyContainer>
+					{
+						canUploadToBackend(status) ? (
+							<>
+								<RevisionsListEmptyText>
+									<FormattedMessage id="drawings.revisions.emptyMessage" defaultMessage="You haven’t added any Files." />
+								</RevisionsListEmptyText>
+								<Button
+									startIcon={<ArrowUpCircleIcon />}
+									variant="contained"
+									color="primary"
+									onClick={() => uploadToDrawing(drawingId)}
+								>
+									<FormattedMessage id="drawings.revisions.uploadFile" defaultMessage="Upload File" />
+								</Button>
+							</>
+						) : (
+							<RevisionsListEmptyText>
+								<FormattedMessage id="drawings.revisions.emptyMessageBusy" defaultMessage="Your files are being processed at this moment, please wait before creating new revisions for this drawing." />
+							</RevisionsListEmptyText>
+						)
+					}
+				</RevisionsListEmptyContainer>
+			</RevisionsListEmptyWrapper>
+		);
+	}
+
+	return (
+		<Container>
+			<RevisionsListHeaderContainer>
+				<RevisionsListHeaderLabel width={140} tabletWidth={94}><FormattedMessage id="drawingRevisionDetails.addedOn" defaultMessage="Added on" /></RevisionsListHeaderLabel>
+				<RevisionsListHeaderLabel width={170} tabletWidth={155}><FormattedMessage id="drawingRevisionDetails.addedBy" defaultMessage="Added by" /></RevisionsListHeaderLabel>
+				<RevisionsListHeaderLabel width={150} tabletWidth={300}><FormattedMessage id="drawingRevisionDetails.statusCode" defaultMessage="Status code" /></RevisionsListHeaderLabel>
+				<RevisionsListHeaderLabel width={150} tabletWidth={300}><FormattedMessage id="drawingRevisionDetails.revisionCode" defaultMessage="Revision code" /></RevisionsListHeaderLabel>
+				<RevisionsListHeaderLabel hideWhenSmallerThan={1140}><FormattedMessage id="drawingRevisionDetails.description" defaultMessage="Description" /></RevisionsListHeaderLabel>
+				<RevisionsListHeaderLabel width={90} tabletWidth={45} hideWhenSmallerThan={800}><FormattedMessage id="drawingRevisionDetails.format" defaultMessage="Format" /></RevisionsListHeaderLabel>
+			</RevisionsListHeaderContainer>
+			<RevisionsList>
+				{isLoading ? (
+					range(min([revisionsCount || 1, 5])).map((key) => <SkeletonListItem key={key} />)
+				) : (
+					revisions.map((revision, i) => (
+						<RevisionsListItemWrapper
+							selected={i === selected}
+							isBeforeSelected={i === selected - 1}
+							onClick={() => {}}
```

Comment:
Can we remove these onClick props that do nothing?

Reply:
It doesn't matter at the moment, to be fair.

Reply:
This also exists in the one for containers. The open revision functionality is done with a Link not an onClick